2012/02/04 16:45:46 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'gmoney'' in Y:\home\localhost\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 Y:\home\localhost\yii\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 Y:\home\localhost\yii\framework\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 Y:\home\localhost\yii\framework\base\CModule.php(388): CDbConnection->init()
#3 Y:\home\localhost\yii\framework\base\CModule.php(104): CModule->getComponent('db')
#4 Y:\home\localhost\yii\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 Y:\home\localhost\yii\framework\web\CFormModel.php(40): ModelCode->init()
#6 Y:\home\localhost\yii\framework\gii\CCodeGenerator.php(152): CFormModel->__construct()
#7 Y:\home\localhost\yii\framework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 Y:\home\localhost\yii\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 Y:\home\localhost\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 Y:\home\localhost\yii\framework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 Y:\home\localhost\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 Y:\home\localhost\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 Y:\home\localhost\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 Y:\home\localhost\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 Y:\home\localhost\yii\gmoney\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/gmoney/index.php?r=gii/model
HTTP_REFERER=http://yii.localhost:8080/gmoney/index.php?r=gii/default/index
---
2012/02/04 18:10:48 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii.accounts' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `accounts`.
in Y:\home\localhost\yii\gmoney\index.php (13)
2012/02/19 13:43:49 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii.tbl_user' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_user`.
in C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\protected\controllers\UserController.php (130)
in C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\index.php (13)
2012/02/19 13:43:49 [error] [exception.CDbException] exception 'CDbException' with message 'The table "tbl_user" for active record class "User" cannot be found in the database.' in C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\db\ar\CActiveRecord.php:2264
Stack trace:
#0 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(User))
#1 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CActiveDataProvider.php(75): CActiveRecord::model('User')
#2 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\protected\controllers\UserController.php(130): CActiveDataProvider->__construct('User')
#3 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#4 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#8 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(292): CFilterChain->run()
#11 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(276): CController->run('')
#13 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#14 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/gmoney/user/
---
2012/02/19 13:46:48 [error] [exception.CException] exception 'CException' with message 'Property "User.prefix" is not defined.' in C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\base\CComponent.php:174
Stack trace:
#0 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\db\ar\CActiveRecord.php(160): CComponent->__set('prefix', 'test2')
#1 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\protected\models\User.php(28): CActiveRecord->__set('prefix', 'test2')
#2 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\db\ar\CActiveRecord.php(1034): User->afterSave()
#3 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#4 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\protected\controllers\UserController.php(72): CActiveRecord->save()
#5 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#6 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#7 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#8 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CFilter.php(41): CFilterChain->run()
#9 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#10 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#11 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#12 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(292): CFilterChain->run()
#13 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(276): CController->run('create')
#15 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('user/create')
#16 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/gmoney/user/create
HTTP_REFERER=http://yii.localhost:8080/gmoney/user/create
---
2012/02/19 16:52:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "user".' in C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php:484
Stack trace:
#0 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(271): CController->missingAction('user')
#1 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(276): CController->run('user')
#2 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('site/user')
#3 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/gmoney/site/user
HTTP_REFERER=http://yii.localhost:8080/gmoney/user
---
2012/02/19 16:52:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "user".' in C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php:484
Stack trace:
#0 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CController.php(271): CController->missingAction('user')
#1 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(276): CController->run('user')
#2 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\web\CWebApplication.php(135): CWebApplication->runController('site/user')
#3 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\Users\Станислав\Documents\My Dropbox\GLoSS\Проекты\gMoney\yii\gmoney\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/gmoney/site/user
HTTP_REFERER=http://yii.localhost:8080/gmoney/user
---
2012/02/19 17:24:52 [error] [exception.CException] exception 'CException' with message 'Property "User.name" is not defined.' in Y:\home\localhost\gmoney\framework\base\CComponent.php:131
Stack trace:
#0 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(144): CComponent->__get('name')
#1 xdebug eval(1): CActiveRecord->__get('name')
#2 Y:\home\localhost\gmoney\gmoney\protected\models\User.php(29): User::beforeSave()
#3 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(1008): User->beforeSave()
#4 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#5 Y:\home\localhost\gmoney\gmoney\protected\controllers\UserController.php(72): CActiveRecord->save()
#6 Y:\home\localhost\gmoney\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#7 Y:\home\localhost\gmoney\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 Y:\home\localhost\gmoney\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 Y:\home\localhost\gmoney\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#11 Y:\home\localhost\gmoney\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 Y:\home\localhost\gmoney\framework\web\CController.php(292): CFilterChain->run()
#14 Y:\home\localhost\gmoney\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(276): CController->run('create')
#16 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(135): CWebApplication->runController('user/create')
#17 Y:\home\localhost\gmoney\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 Y:\home\localhost\gmoney\gmoney\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/gmoney/user/create
HTTP_REFERER=http://gmoney.localhost:8080/gmoney/user/create
---
2012/02/19 17:32:11 [error] [exception.CException] exception 'CException' with message 'Property "User.name" is not defined.' in Y:\home\localhost\gmoney\framework\base\CComponent.php:131
Stack trace:
#0 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(144): CComponent->__get('name')
#1 xdebug eval(1): CActiveRecord->__get('name')
#2 Y:\home\localhost\gmoney\gmoney\protected\models\User.php(32): User::afterSave()
#3 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(1034): User->afterSave()
#4 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#5 Y:\home\localhost\gmoney\gmoney\protected\controllers\UserController.php(72): CActiveRecord->save()
#6 Y:\home\localhost\gmoney\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#7 Y:\home\localhost\gmoney\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 Y:\home\localhost\gmoney\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 Y:\home\localhost\gmoney\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#11 Y:\home\localhost\gmoney\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 Y:\home\localhost\gmoney\framework\web\CController.php(292): CFilterChain->run()
#14 Y:\home\localhost\gmoney\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(276): CController->run('create')
#16 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(135): CWebApplication->runController('user/create')
#17 Y:\home\localhost\gmoney\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 Y:\home\localhost\gmoney\gmoney\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/gmoney/user/create
HTTP_REFERER=http://gmoney.localhost:8080/gmoney/user/create
---
2012/02/19 17:36:01 [error] [exception.CException] exception 'CException' with message 'Property "User.name" is not defined.' in Y:\home\localhost\gmoney\framework\base\CComponent.php:131
Stack trace:
#0 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(144): CComponent->__get('name')
#1 xdebug eval(1): CActiveRecord->__get('name')
#2 Y:\home\localhost\gmoney\gmoney\protected\models\User.php(29): User::afterSave()
#3 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(1034): User->afterSave()
#4 Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php(787): CActiveRecord->insert(NULL)
#5 Y:\home\localhost\gmoney\gmoney\protected\controllers\UserController.php(72): CActiveRecord->save()
#6 Y:\home\localhost\gmoney\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#7 Y:\home\localhost\gmoney\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 Y:\home\localhost\gmoney\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 Y:\home\localhost\gmoney\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#11 Y:\home\localhost\gmoney\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 Y:\home\localhost\gmoney\framework\web\CController.php(292): CFilterChain->run()
#14 Y:\home\localhost\gmoney\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(276): CController->run('create')
#16 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(135): CWebApplication->runController('user/create')
#17 Y:\home\localhost\gmoney\framework\base\CApplication.php(162): CWebApplication->processRequest()
#18 Y:\home\localhost\gmoney\gmoney\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/gmoney/user/create
HTTP_REFERER=http://gmoney.localhost:8080/gmoney/user/create
---
2012/02/19 19:10:14 [error] [exception.CException] exception 'CException' with message 'UserController and its behaviors do not have a method or closure named "getIsNewRecord".' in Y:\home\localhost\gmoney\framework\base\CComponent.php:266
Stack trace:
#0 xdebug eval(1): CComponent->__call('getIsNewRecord', Array)
#1 xdebug eval(1): UserController->getIsNewRecord()
#2 Y:\home\localhost\gmoney\gmoney\protected\controllers\UserController.php(130): UserController::actionIndex()
#3 Y:\home\localhost\gmoney\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#4 Y:\home\localhost\gmoney\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 Y:\home\localhost\gmoney\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 Y:\home\localhost\gmoney\framework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#8 Y:\home\localhost\gmoney\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 Y:\home\localhost\gmoney\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 Y:\home\localhost\gmoney\framework\web\CController.php(292): CFilterChain->run()
#11 Y:\home\localhost\gmoney\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(276): CController->run('')
#13 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(135): CWebApplication->runController('user')
#14 Y:\home\localhost\gmoney\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 Y:\home\localhost\gmoney\gmoney\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/gmoney/user
HTTP_REFERER=http://gmoney.localhost:8080/gmoney/?XDEBUG_SESSION_START=netbeans
---
2012/02/20 16:22:51 [error] [php] Missing argument 2 for AccountE::__construct(), called in Y:\home\localhost\gmoney\framework\db\ar\CActiveRecord.php on line 378 and defined (Y:\home\localhost\gmoney\gmoney\protected\models\AccountE.php:15)
Stack trace:
#0 Y:\home\localhost\gmoney\framework\validators\CInlineValidator.php(43): CrudCode->validateModel()
#1 Y:\home\localhost\gmoney\framework\validators\CValidator.php(197): CInlineValidator->validateAttribute()
#2 Y:\home\localhost\gmoney\framework\base\CModel.php(158): CInlineValidator->validate()
#3 Y:\home\localhost\gmoney\framework\gii\CCodeGenerator.php(158): CrudCode->validate()
#4 Y:\home\localhost\gmoney\framework\gii\CCodeGenerator.php(64): CrudGenerator->prepare()
#5 Y:\home\localhost\gmoney\framework\web\actions\CInlineAction.php(50): CrudGenerator->actionIndex()
#6 Y:\home\localhost\gmoney\framework\web\CController.php(309): CInlineAction->runWithParams()
#7 Y:\home\localhost\gmoney\framework\web\CController.php(287): CrudGenerator->runAction()
#8 Y:\home\localhost\gmoney\framework\web\CController.php(266): CrudGenerator->runActionWithFilters()
#9 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(276): CrudGenerator->run()
#10 Y:\home\localhost\gmoney\framework\web\CWebApplication.php(135): CWebApplication->runController()
#11 Y:\home\localhost\gmoney\framework\base\CApplication.php(162): CWebApplication->processRequest()
#12 Y:\home\localhost\gmoney\gmoney\index.php(13): CWebApplication->run()
REQUEST_URI=/gmoney/gii/crud
in Y:\home\localhost\gmoney\gmoney\protected\models\AccountE.php (15)
in Y:\home\localhost\gmoney\gmoney\index.php (13)
2012/02/20 16:50:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 16:50:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 17:11:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 17:12:12 [error] [php] Missing argument 2 for AccountE::__construct(), called in Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php on line 378 and defined (Y:\home\localhost\gmoney\protected\models\AccountE.php:15)
Stack trace:
#0 Y:\home\localhost\yiiframework\validators\CInlineValidator.php(43): CrudCode->validateModel()
#1 Y:\home\localhost\yiiframework\validators\CValidator.php(197): CInlineValidator->validateAttribute()
#2 Y:\home\localhost\yiiframework\base\CModel.php(158): CInlineValidator->validate()
#3 Y:\home\localhost\yiiframework\gii\CCodeGenerator.php(158): CrudCode->validate()
#4 Y:\home\localhost\yiiframework\gii\CCodeGenerator.php(64): CrudGenerator->prepare()
#5 Y:\home\localhost\yiiframework\web\actions\CInlineAction.php(50): CrudGenerator->actionIndex()
#6 Y:\home\localhost\yiiframework\web\CController.php(309): CInlineAction->runWithParams()
#7 Y:\home\localhost\yiiframework\web\CController.php(287): CrudGenerator->runAction()
#8 Y:\home\localhost\yiiframework\web\CController.php(266): CrudGenerator->runActionWithFilters()
#9 Y:\home\localhost\yiiframework\web\CWebApplication.php(276): CrudGenerator->run()
#10 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController()
#11 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#12 Y:\home\localhost\gmoney\index.php(13): CWebApplication->run()
REQUEST_URI=/gii/crud
in Y:\home\localhost\gmoney\protected\models\AccountE.php (15)
in Y:\home\localhost\gmoney\index.php (13)
2012/02/20 17:12:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 17:12:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 17:49:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 17:49:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 17:49:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 17:49:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:08:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:08:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:09:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:09:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:09:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:09:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:09:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:09:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/20 21:09:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:42:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:43:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:44:08 [warning] [application] Failed to set unsafe attribute "prefix" of "User".
in Y:\home\localhost\gmoney\protected\controllers\UserController.php (71)
in Y:\home\localhost\gmoney\index.php (13)
2012/02/21 17:44:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:44:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:44:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:45:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:45:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:45:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:45:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 17:51:45 [error] [exception.CException] exception 'CException' with message 'CWebUser and its behaviors do not have a method or closure named "getPrefix".' in Y:\home\localhost\yiiframework\base\CComponent.php:266
Stack trace:
#0 Y:\home\localhost\gmoney\protected\views\user\index.php(18): CComponent->__call('getPrefix', Array)
#1 Y:\home\localhost\gmoney\protected\views\user\index.php(18): CWebUser->getPrefix()
#2 Y:\home\localhost\yiiframework\web\CBaseController.php(127): require('Y:\home\localho...')
#3 Y:\home\localhost\yiiframework\web\CBaseController.php(96): CBaseController->renderInternal('Y:\home\localho...', Array, true)
#4 Y:\home\localhost\yiiframework\web\CController.php(870): CBaseController->renderFile('Y:\home\localho...', Array, true)
#5 Y:\home\localhost\yiiframework\web\CController.php(783): CController->renderPartial('index', Array, true)
#6 Y:\home\localhost\gmoney\protected\controllers\UserController.php(133): CController->render('index', Array)
#7 Y:\home\localhost\yiiframework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#8 Y:\home\localhost\yiiframework\web\CController.php(309): CInlineAction->runWithParams(Array)
#9 Y:\home\localhost\yiiframework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#10 Y:\home\localhost\yiiframework\web\filters\CFilter.php(41): CFilterChain->run()
#11 Y:\home\localhost\yiiframework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#12 Y:\home\localhost\yiiframework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#13 Y:\home\localhost\yiiframework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#14 Y:\home\localhost\yiiframework\web\CController.php(292): CFilterChain->run()
#15 Y:\home\localhost\yiiframework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 Y:\home\localhost\yiiframework\web\CWebApplication.php(276): CController->run('')
#17 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('user')
#18 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#19 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/user
HTTP_REFERER=http://gmoney.localhost:8080/index.php
---
2012/02/21 17:51:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:02:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:03:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:03:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:03:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:03:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:06:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:06:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:06:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:06:36 [error] [exception.Exception] exception 'Exception' with message 'The prefix is not defined.' in Y:\home\localhost\gmoney\protected\models\AccountE.php:30
Stack trace:
#0 Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php(2262): AccountE->tableName()
#1 Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(AccountE))
#2 Y:\home\localhost\yiiframework\gii\generators\crud\CrudCode.php(66): CActiveRecord::model('AccountE')
#3 Y:\home\localhost\yiiframework\validators\CInlineValidator.php(43): CrudCode->validateModel('model', Array)
#4 Y:\home\localhost\yiiframework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#5 Y:\home\localhost\yiiframework\base\CModel.php(158): CValidator->validate(Object(CrudCode), NULL)
#6 Y:\home\localhost\yiiframework\gii\CCodeGenerator.php(158): CModel->validate()
#7 Y:\home\localhost\yiiframework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 Y:\home\localhost\yiiframework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 Y:\home\localhost\yiiframework\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 Y:\home\localhost\yiiframework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 Y:\home\localhost\yiiframework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 Y:\home\localhost\yiiframework\web\CWebApplication.php(276): CController->run('')
#13 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('gii/crud')
#14 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#15 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/gii/crud
HTTP_REFERER=http://gmoney.localhost:8080/gii/crud
---
2012/02/21 18:06:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:07:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:07:24 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii.user1' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `user1`.
in Y:\home\localhost\gmoney\index.php (13)
2012/02/21 18:07:24 [error] [exception.CDbException] exception 'CDbException' with message 'The table "user1" for active record class "AccountE" cannot be found in the database.' in Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php:2264
Stack trace:
#0 Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(AccountE))
#1 Y:\home\localhost\yiiframework\gii\generators\crud\CrudCode.php(66): CActiveRecord::model('AccountE')
#2 Y:\home\localhost\yiiframework\validators\CInlineValidator.php(43): CrudCode->validateModel('model', Array)
#3 Y:\home\localhost\yiiframework\validators\CValidator.php(197): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#4 Y:\home\localhost\yiiframework\base\CModel.php(158): CValidator->validate(Object(CrudCode), NULL)
#5 Y:\home\localhost\yiiframework\gii\CCodeGenerator.php(158): CModel->validate()
#6 Y:\home\localhost\yiiframework\gii\CCodeGenerator.php(64): CCodeGenerator->prepare()
#7 Y:\home\localhost\yiiframework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#8 Y:\home\localhost\yiiframework\web\CController.php(309): CInlineAction->runWithParams(Array)
#9 Y:\home\localhost\yiiframework\web\CController.php(287): CController->runAction(Object(CInlineAction))
#10 Y:\home\localhost\yiiframework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 Y:\home\localhost\yiiframework\web\CWebApplication.php(276): CController->run('')
#12 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('gii/crud')
#13 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#14 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/gii/crud
HTTP_REFERER=http://gmoney.localhost:8080/gii/crud
---
2012/02/21 18:07:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:07:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:07:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:07:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:07:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:08:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:08:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:08:31 [warning] [application] Failed to set unsafe attribute "start_amount" of "AccountE".
in Y:\home\localhost\gmoney\protected\controllers\AccountEController.php (71)
in Y:\home\localhost\gmoney\index.php (13)
2012/02/21 18:08:31 [warning] [application] Failed to set unsafe attribute "date_open" of "AccountE".
in Y:\home\localhost\gmoney\protected\controllers\AccountEController.php (71)
in Y:\home\localhost\gmoney\index.php (13)
2012/02/21 18:08:31 [error] [exception.CException] exception 'CException' with message 'Property "AccountE.code" is not defined.' in Y:\home\localhost\yiiframework\base\CComponent.php:131
Stack trace:
#0 Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php(144): CComponent->__get('code')
#1 Y:\home\localhost\yiiframework\validators\CStringValidator.php(80): CActiveRecord->__get('code')
#2 Y:\home\localhost\yiiframework\validators\CValidator.php(197): CStringValidator->validateAttribute(Object(AccountE), 'code')
#3 Y:\home\localhost\yiiframework\base\CModel.php(158): CValidator->validate(Object(AccountE), NULL)
#4 Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php(786): CModel->validate(NULL)
#5 Y:\home\localhost\gmoney\protected\controllers\AccountEController.php(72): CActiveRecord->save()
#6 Y:\home\localhost\yiiframework\web\actions\CInlineAction.php(50): AccountEController->actionCreate()
#7 Y:\home\localhost\yiiframework\web\CController.php(309): CInlineAction->runWithParams(Array)
#8 Y:\home\localhost\yiiframework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 Y:\home\localhost\yiiframework\web\filters\CFilter.php(41): CFilterChain->run()
#10 Y:\home\localhost\yiiframework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#11 Y:\home\localhost\yiiframework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 Y:\home\localhost\yiiframework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 Y:\home\localhost\yiiframework\web\CController.php(292): CFilterChain->run()
#14 Y:\home\localhost\yiiframework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 Y:\home\localhost\yiiframework\web\CWebApplication.php(276): CController->run('create')
#16 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('accountE/create')
#17 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#18 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/accountE/create
HTTP_REFERER=http://gmoney.localhost:8080/accountE/create
---
2012/02/21 18:08:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:09:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:10:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:11:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:11:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:12:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:12:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:16:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:16:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:16:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:16:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:16:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:16:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:16:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:17:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:17:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:17:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:17:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:17:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:17:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 18:17:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:19:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:19:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:20:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:20:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:20:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:21:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:21:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:21:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:21:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:21:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:22:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:22:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:22:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:22:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:24:14 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii.$prefix_income_categories' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `$prefix_income_categories`.
in Y:\home\localhost\gmoney\protected\controllers\IncomeCategoryController.php (130)
in Y:\home\localhost\gmoney\index.php (13)
2012/02/21 19:24:14 [error] [exception.CDbException] exception 'CDbException' with message 'The table "$prefix_income_categories" for active record class "IncomeCategory" cannot be found in the database.' in Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php:2264
Stack trace:
#0 Y:\home\localhost\yiiframework\db\ar\CActiveRecord.php(379): CActiveRecordMetaData->__construct(Object(IncomeCategory))
#1 Y:\home\localhost\yiiframework\web\CActiveDataProvider.php(75): CActiveRecord::model('IncomeCategory')
#2 Y:\home\localhost\gmoney\protected\controllers\IncomeCategoryController.php(130): CActiveDataProvider->__construct('IncomeCategory')
#3 Y:\home\localhost\yiiframework\web\actions\CInlineAction.php(50): IncomeCategoryController->actionIndex()
#4 Y:\home\localhost\yiiframework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 Y:\home\localhost\yiiframework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 Y:\home\localhost\yiiframework\web\filters\CFilter.php(41): CFilterChain->run()
#7 Y:\home\localhost\yiiframework\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#8 Y:\home\localhost\yiiframework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 Y:\home\localhost\yiiframework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 Y:\home\localhost\yiiframework\web\CController.php(292): CFilterChain->run()
#11 Y:\home\localhost\yiiframework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 Y:\home\localhost\yiiframework\web\CWebApplication.php(276): CController->run('')
#13 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('IncomeCategory')
#14 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#15 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/IncomeCategory
---
2012/02/21 19:24:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/21 19:24:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 19:59:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 19:59:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:00:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:13:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:13:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:19:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:23:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:24:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:24:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:24:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:29:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "assets/e7f449b6/css/images/arrows-ffffff.png".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('assets/e7f449b6...')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/assets/e7f449b6/css/images/arrows-ffffff.png
HTTP_REFERER=http://gmoney.localhost:8080/expenseCategory
---
2012/02/22 20:29:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "assets/e7f449b6/css/images/shadow.png".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('assets/e7f449b6...')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/assets/e7f449b6/css/images/shadow.png
HTTP_REFERER=http://gmoney.localhost:8080/expenseCategory
---
2012/02/22 20:29:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:44:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:53:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/22 20:58:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/23 06:35:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:46:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:47:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:48:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:49:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:49:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:49:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:51:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:51:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:51:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:51:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:51:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "operation".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('operation')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/operation
HTTP_REFERER=http://gmoney.localhost:8080/user
---
2012/02/26 10:52:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:52:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:53:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:53:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:53:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 10:53:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 11:21:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 12:31:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 12:31:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "operation".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('operation')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/operation
HTTP_REFERER=http://gmoney.localhost:8080/expenseCategory
---
2012/02/26 12:31:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2012/02/26 12:31:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Y:\home\localhost\yiiframework\web\CWebApplication.php:280
Stack trace:
#0 Y:\home\localhost\yiiframework\web\CWebApplication.php(135): CWebApplication->runController('favicon.ico')
#1 Y:\home\localhost\yiiframework\base\CApplication.php(162): CWebApplication->processRequest()
#2 Y:\home\localhost\gmoney\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
